flink-ml-iteration/src/main/java/org/apache/flink/iteration/datacache/nonkeyed/Segment.java

```diff
@@ -18,38 +18,49 @@
 
 package org.apache.flink.iteration.datacache.nonkeyed;
 
+import org.apache.flink.api.common.typeutils.TypeSerializer;
 import org.apache.flink.core.fs.Path;
 
+import java.io.IOException;
 import java.io.Serializable;
+import java.util.List;
 import java.util.Objects;
 
-/** A segment represents a single file for the cache. */
+/** A segment contains the information about a cache unit. */
```

Comment:
After this PR, the Java doc says this class represents a cache unit. But the implementation shows that this class could represent a file, which seems confusing. Could you clarify it?

Reply:
This class contains all information about a cache unit, no matter the cache is on disk, in memory, or both. I'll modify the doc and the api to improve its readability.

Reply:
This class contains all information about a cache unit, no matter the cache is on disk, in memory, or both. I'll modify the doc and the api to improve its readability.